hail_scripts/reference_data/config.py

```diff
@@ -401,6 +396,27 @@ def custom_mpc_select(ht):
             },
         },
     },
+    'dbnsfp_mito': {
```

Comment:
Just a heads up that I opted to rename the field from `dbnsfp` to `dbnsfp_mito` and separate the configs.  Keeping a single config got too hairy with different selects.

Reply:
I like this. It also makes it easier for seqr to handle, as we can be explicit about which reference source we are pulling from